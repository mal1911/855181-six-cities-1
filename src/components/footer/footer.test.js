import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import Footer from "./footer";

it(`Footer correctly renders`, () => {
  const tree = renderer
    .create(<BrowserRouter>
      <Footer/>
    </BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
