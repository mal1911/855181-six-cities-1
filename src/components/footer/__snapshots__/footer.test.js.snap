// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer correctly renders 1`] = `
<footer
  className="footer container"
>
  <a
    aria-current="page"
    className="footer__logo-link active"
    href="/"
    onClick={[Function]}
    style={Object {}}
  >
    <img
      alt="6 cities logo"
      className="footer__logo"
      height="33"
      src="/img/logo.svg"
      width="64"
    />
  </a>
</footer>
`;
